Website/src/rendering/src/components/Breadcrumb.tsx

```diff
@@ -1,16 +1,86 @@
 import React from 'react';
-import Breadcrumbs from 'nextjs-breadcrumbs';
+import { ComponentProps } from 'lib/component-props';
+
+function formatBreadcrumb(text: string) {
+  text = decodeURIComponent(text);
+  return text;
+}
+
+type contextItem = {
+  url: { path: string };
+  name: string;
+  pageTitle: { value: string };
+  displayName: string;
+  title: { value: string };
+  ancestors: contextItem[];
+};
+
+type BreadcrumbProps = ComponentProps & {
+  fields: {
+    data: {
+      item: contextItem;
+      ancestors: contextItem[];
+    };
+  };
+};
+
+function getBreadcrumbTitle(propItem: contextItem) {
```

Comment:
![image](https://user-images.githubusercontent.com/1315023/130799719-7539b404-d065-41c4-8d81-07570d3f2887.png)

1. Play Summit! Is repeated multiple times. 
2. The first item should probably be "Home"?

Reply:
these are being pulled from the page title field. xm values need to be updated. scott is on that task. 
if no page title the fallbacks are in place as discussed and will look good